### PR TITLE
Update sync docs

### PR DESCRIPTION
**Pull request by prees1**
- simplify wording
- include info about creation of custom fields and different behaviour
  depending on command usage.

### CODE REVIEW

**Comment by prees1** [comment]
can you review please @SamDesmondKing ?

File: pages/tidal tools/syncservers.md [hunk 1]
```diff
@@ -8,29 +8,31 @@ permalink: sync-servers.html
...
 This data can be passed as standard input to the `tidal sync servers` command and your servers data will be
 synchronized via the API.
 
-
 The JSON document must be created in the specified format. The top-level key must be `"servers"`,
 with a value of an array. The array can consist of the various keys as shown below, describing the server to be synced.
 You can also include any other arbitrary fields in the key `"custom_fields"`.
 
+{% include note.html content="If you pass data to Tidal Tools via standard input (ie. `cat x.json | tidal sync servers`)
+any custom fields presented will be automatically created if they don't already exist. If you provide the data to the
```
Comment (SamDesmondKing) [inline pages/tidal tools/syncservers.md:32]: I'll have to take your word on this, most of our experience in this area was MS accuracy / limitation tests, we didn't get too close to the custom field creation logic. I remember there was some confusion around the docs on this last week right @jmariomejiap?

File: pages/tidal tools/syncservers.md [hunk 2]
```diff
@@ -8,29 +8,31 @@ permalink: sync-servers.html
...
-If the given server to sync does not exist already, it will add that server to the Tidal Migrations platform.
+Syncing is a process that creates and updates your inventories to your Tidal Migrations Workspace. This is useful if you
+would like to update a lot of data across many different records. It can also be useful to keep your inventory up to date with an external
+date source, by continually updating your inventory on a recurring scheudle.
+
 
+## How do I sync my servers?
 
-## How do I manually sync my servers?
+{% include note.html content="You will first need to have [Tidal Tools installed and login](https://guides.tidalmg.com/tidal-tools.html) to your Workspace." %}
```
Comment (SamDesmondKing) [inline pages/tidal tools/syncservers.md:20]:
```suggestion
{% include note.html content="You will first need to have [Tidal Tools installed and logged in](https://guides.tidalmg.com/tidal-tools.html) to your Workspace." %}
```

File: pages/tidal tools/syncservers.md [hunk 3]
```diff
@@ -8,29 +8,31 @@ permalink: sync-servers.html
...
-After having [installed](tidal-tools.html#install) Tidal Tools, begin to sync your inventories with `tidal sync servers`. Tidal sync supports many server inventory management tools such as
-VMWare, HyperV, and more.
-
 ## What is Syncing?
 
-Syncing is a process that transfers your inventories to your Tidal Migrations account.
-When importing your servers to the API, Tidal Migrations's sync tool will check for existing servers, based on their hostname
-and update the changed data for those servers.
-If the given server to sync does not exist already, it will add that server to the Tidal Migrations platform.
+Syncing is a process that creates and updates your inventories to your Tidal Migrations Workspace. This is useful if you
```
Comment (SamDesmondKing) [inline pages/tidal tools/syncservers.md:13]:
```suggestion
Syncing is a process that creates and updates your inventories in your Tidal Migrations Workspace. This is useful if you
```

... I think?

File: pages/tidal tools/syncservers.md [hunk 4]
```diff
@@ -8,29 +8,31 @@ permalink: sync-servers.html
...
+would like to update a lot of data across many different records. It can also be useful to keep your inventory up to date with an external
+date source, by continually updating your inventory on a recurring scheudle.
```
Comment (SamDesmondKing) [inline pages/tidal tools/syncservers.md:15]:
```suggestion
date source, by continually updating your inventory on a recurring schedule.
```

File: pages/tidal tools/syncservers.md [hunk 5]
```diff
@@ -69,6 +71,8 @@ The synchronization of your servers to Tidal Migrations can be performed with th
 tidal sync servers some_file.json
 ```
 
+Any servers provided that already exist, based on their hostname, will be updated with the data
+provided in the JSON document. If no server with the given hostname exist, that server will be created with all the included data.
```
Comment (SamDesmondKing) [inline pages/tidal tools/syncservers.md:75]:
```suggestion
provided in the JSON document. If no server with the given hostname exists, that server will be created with all the included data.
```

File: pages/tidal tools/syncservers.md [hunk 6]
```diff
@@ -69,6 +71,8 @@ The synchronization of your servers to Tidal Migrations can be performed with th
 tidal sync servers some_file.json
 ```
 
+Any servers provided that already exist, based on their hostname, will be updated with the data
```
Comment (SamDesmondKing) [inline pages/tidal tools/syncservers.md:74]: Nice 👍

**Review by SamDesmondKing — APPROVED** [review]
Thanks for adding this Phil. Left some comments and suggestions :)

File: pages/tidal tools/syncservers.md (hunk 1 above)
Comment (justinbarclay) [reply pages/tidal tools/syncservers.md:32]: Does the CLI do this? I don't remember the backend creating new fields.

**Pull request merged**